pom.xml

```diff
@@ -52,6 +55,40 @@
     <plugin.maven.enforcer.version>3.0.0</plugin.maven.enforcer.version>
   </properties>
 
+  <profiles>
+    <profile>
+      <id>release</id>
```

Comment:
I'd suggest adding a note in the documentation to call out the need to activate the release profile when performing a release (`mvn deploy -P release`)

Reply:
do we need a new documentation section for this parent project?

Reply:
i have added a parent pom documentation page under dev including this and other information. we should remember to update this page with any important changes we make to the pom